less/_grid.less

```diff
@@ -188,11 +188,12 @@
 	}
 	// when the page has a max-width set
 	&.max-width {
-		@max-width: 1280px;
```

Comment:
not sure why this was here, it was preventing the value from the skin.less file in the interface from being used...